File: fieldsync-inventory-backend/src/main/java/com/example/fieldsync_inventory_api/dto/inventory/season/SeasonResponseDTO.java
package com.example.fieldsync_inventory_api.dto.inventory.season;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.Instant;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class SeasonResponseDTO {
    private Integer id;
    private String name;
    private String description;
    private Instant createdAt;
    private Instant updatedAt;
    private Instant deletedAt;
}

File: fieldsync-inventory-app/app/src/main/java/com/example/fieldsync_inventory_app/data/remote/dto/season/SeasonResponseDto.kt
package com.example.fieldsync_inventory_app.data.remote.dto.season

// SeasonResponseDto for the server's response
data class SeasonResponseDto(
    val id: Int,
    val name: String,
    val description: String,
    val createdAt: String,
    val updatedAt: String,
    val deletedAt: String?
)
